src/Microsoft.Extensions.HealthChecks/HealthCheckServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Provides extension methods for registering <see cref="IHealthCheckService"/> in an <see cref="IServiceCollection"/>.
+    /// </summary>
+    public static class HealthCheckServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds the <see cref="IHealthCheckService"/> to the container, using the provided delegate to register
+        /// health checks.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to add the <see cref="IHealthCheckService"/> to.</param>
+        /// <param name="configure">A delegate which recieves a <see cref="IHealthChecksBuilder"/> and registers health checks on it.</param>
+        /// <returns>The original <see cref="IServiceCollection"/> instance</returns>
+        public static IServiceCollection AddHealthChecks(this IServiceCollection services, Action<IHealthChecksBuilder> configure)
```

Comment:
It would be a dope to have one of these without the configure. Given that other components might inject checks it seems useful

Reply:
Yeah, I literally had that and yanked it before the first push. I thought it seemed kinda useless at the time because why would you add health checks but no checks. Your point about adding it but relying on other components to add checks makes some sense, especially since other components may not want to depend upon `Microsoft.Extensions.HealthChecks` so they won't be using `.AddHealthChecks` to add their checks.